Code/Framework/AzFramework/Platform/Windows/AzFramework/Input/Devices/Mouse/InputDeviceMouse_Windows.cpp

```diff
@@ -7,6 +7,7 @@
  */
 
 #include <AzCore/PlatformIncl.h>
+#include <AzCore/Module/Environment.h>
```

Comment:
Does InputDeviceKeyboard_Windows.cpp also need this #include?

Reply:
Yes, added.